lib/components/storyblok-component.tsx

```diff
@@ -1,15 +1,25 @@
 import React from "react";
+import type { SbBlokData } from "../types";
 import { getComponent } from "../index";
 
-const StoryblokComponent = ({ blok, ...restProps }) => {
+interface StoryblokComponentProps {
+  blok: SbBlokData | false;
+  restProps?: any;
+}
+
+const StoryblokComponent = ({
+  blok,
+  ...restProps
+}: StoryblokComponentProps): JSX.Element => {
```

Comment:
Been checking this article from Kent and looks like the way you did it it's the best way to type a React component, good choice! https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript